core/tests/histogram_test.cc

```diff
@@ -11,7 +11,7 @@ TEST(HistogramTest, initialize_with_zero) {
   Histogram histogram{{}};
   auto metric = histogram.Collect();
   auto h = metric.histogram;
-  EXPECT_EQ(h.sample_count, 0);
```

Comment:
`EXPECT_EQ(h.sample_count, 0U);`  ---> EXPECT_EQ(0U, h.sample_count);
IIRC, the 1st argument of the function `EXPECT_EQ` means the expected value, and the 2nd one is real value.

Reply:
I only fix the compiler warnings and preserve the order as it was.